src/System.Reflection/tests/TestExe/Program.cs

```diff
@@ -0,0 +1,19 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+
+namespace System.Reflection.Tests
+{
+    public class TestExeException : Exception
+    {
+    }
+
+    public static class TestExe
+    {
+        private static void Main()
+        {
+            throw new TestExeException();
+        }
```

Comment:
I switched it to int32, not sure why I had void on the brain. I don't think it's too bad to only test one signature. I'd have to create 1 test exe per signature, which seems like overkill.